src/pages/PublicProfilePage/PublicProfilePage.tsx

```
@@ -0,0 +1,137 @@
+                    </div>
+                    <div className="flex items-center gap-2">
+                      <Clock size={14} /> Last seen 1 hour ago
+                    </div>
+                    <div className="flex items-center gap-2">
+                      <MessageCircle size={14} /> Avg. response within 10 hours
+                    </div>
+                  </div>
+                </div>
+              </div>
+
+              <button className="text-muted-foreground hover:text-neutral-foreground">
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)